### DISCUSSION TITLE
Welcome to LeGrandMechantLoop Discussions!

### DISCUSSION THREAD

**Discussion by benkuper** [benkuper/LeGrandMechantLoop#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rom1Constant** [comment]
Ouiiii bonjour ben alias Mr le grand méchant boucle !!

J'ai eu le plaisir (et le temps) de me plonger dans LGML pour tester des configs et j'aurais quelques questions dans le désordre :
Avant toute chose bravo, je pense que pas mal de copains.pines musiciens.nes seront intéressés.ées par ton soft en substitut à Live qui peut faire peur à des novices et/ou anti logiciels propriétaires. 
- Est-ce que tu m'autorises à le partager si je rencontre des profils qui cherche ce genre d'outils ?
Quelques questions techniques que j'ai pas vu ou qu'il n'existe pas encore ?
- Il y a t-il un métronome pour que je puisse entendre le bpm ?
- J'ai pas trouvé la possibilité d'avoir un dossier personnalisé pour redirigé vers mes vst (audio fx) et mes vsti (synthétiseurs) ?
- Peut-on intégrer des boucles de ma librairie de samples pour jouer avec (genre drum) ?
- Est-ce que les boutons peuvent être triggé par une value du soft ? 
               Je vois que les slider peuvent être en mode control expression/référence avec une analyzer value mais pas le bouton play loop (par exemple)
- Pourquoi pas intégrer le protocol Link de live pour créer des live looping de folies ??
- Peut-on travailler sur plusieurs cartes son en même temps (un peu comme voice meeter) ?
- Voir est-ce que LGML peux être vu comme une carte son virtuel asio par d'autre soft ? pour concurrencer voice meeter et loopback ^^

Merciiii beaucouuuup et à très bientôt
bises
